fog/view/server/src/sharding_strategy.rs

```diff
@@ -140,4 +167,80 @@ mod epoch_sharding_strategy_tests {
 
         assert!(!should_process_block)
     }
+
+    #[test]
+    fn is_ready_to_serve_tx_outs_first_epoch_processed_block_count_is_zero_less_than_50_percent_processed_returns_true(
```

Comment:
What is this even testing? That zero is less than half of 100?

Reply:
No, that the first block returns a ready status even if < 50% of blocks are processed.